src/components/pacbio/PacbioRunWellEdit.vue

```diff
@@ -41,7 +41,7 @@
           v-model="currentWell.generate_hifi"
           data-attribute="generate-hifi"
           :options="generateHifiOptions[currentRun.system_name]"
-          @change="updateCCSAnalysisOutput"
+          @chaiinputnputnge="updateCCSAnalysisOutput"
```

Comment:
not sure what this is?

Reply:
I think this is just suppose to be @input

Reply:
yes 👍